app/models/unit.rb

```diff
@@ -2676,6 +2676,11 @@ def send_weekly_status_emails(summary_stats)
     return unless send_notifications
 
     summary_stats[:unit] = self
+    
+    #The Line bellow is meant to get the task due date through the task definition from the task defenition in the unit definition
+
+    #summary_stats[:due] = task_definition.due_date
+
```

Comment:
I am not sure that these comments add much to this file. I suggest you remove these.